<h4>Qiime stuff</h4>
<p>.ab1 files are trace files, seq files are the sequence of nucleotides from the trace files</p>
<p>Display and visually clean the trace files looking for N on both ends, trim and do blast search</p>
tool: https://www.youtube.com/watch?v=Xqs6bcYSv34
<p>a1-p1.ab1 too small</p>
<p>a1-p2.ab1 </p>
<h4>instructions for processing using Mega</h4>h4>
<p>1) open a1-p2.ab1</p>
<p>2) find the first N, click to select</p>
<img src="sangera1p1/a1p2_a.png">
<p>once the first N is selected, select mask upstream and the sequence to the left should grey out. 
Click the >>N arrow to select the next N </p>
<img src="sangera1p1/a1p2_b.png">
<img src="sangera1p1/a1p2_b1.png">
<p>this leads to the next N. Select mask downstream and verify the part of the sequence after the N is greyed out.</p>
<img src="sangera1p1/a1p2_c.png">
<img src="sangera1p1/a1p2_c1.png">
<p>Copy the sequence and paste into blast. https://blast.ncbi.nlm.nih.gov/Blast.cgi</p>
Nucleotide search >> rRNA/ITS databases. 
<img src="sangera1p1/a1p2_d.png">
<h4>CodonCode Aligner</h4>
<p>CCA incorporates phred into the eval version. Phred is supposed to outperform the ABI sw by using "simple fourier" analysis. not clear</p>
<p>When reading you get blank, highest quality, light green, dark green lowest quality</p>
<img src="codoncode/codoncode1a.png">
<p>assemble ax-px into a contig</p>p>
<img src="codoncode/codoncode1b.png">
<p>can build a tree on ax-px.ab1</p>
<img src="codoncode/codoncode1c.png">
<h6>phred has to read and clean; match with above MEGA output. </h6>
<img src="codoncode/phred1.png">
<img src="codoncode/phred2.png">
<img src="codoncode/phred3.png">
<img src="codoncode/phred4.png">
<img src="codoncode/phred5.png">
<img src="codoncode/phred6.png">
<img src="codoncode/phred7.png">
<img src="codoncode/phred8.png">
<img src="codoncode/phred9.png">
<img src="codoncode/phred10.png">
<img src="codoncode/phred11.png">
<img src="codoncode/phred12.png">
<img src="codoncode/phred13.png">
<img src="codoncode/phred14.png">


